packages/react/src/UnorderedList/UnorderedList.tsx

```diff
@@ -10,18 +10,27 @@ import { UnorderedListItem } from './UnorderedListItem'
 
 export type UnorderedListProps = {
   markers?: boolean
+  inverseColor?: boolean
 } & PropsWithChildren<HTMLAttributes<HTMLUListElement>>
 
 type UnorderedListComponent = {
   Item: typeof UnorderedListItem
 } & ForwardRefExoticComponent<UnorderedListProps & RefAttributes<HTMLUListElement>>
 
 export const UnorderedList = forwardRef(
-  ({ children, markers = true, className, ...restProps }: UnorderedListProps, ref: ForwardedRef<HTMLUListElement>) => {
+  (
+    { children, markers = true, inverseColor, className, ...restProps }: UnorderedListProps,
```

Comment:
This is only a reorder.

```suggestion
    { children, className, inverseColor, markers = true, ...restProps }: UnorderedListProps,
```